ui/src/components/credentials/LinkDetails.tsx

```diff
@@ -0,0 +1,250 @@
+          fetchJsonSchemaFromUrl({ link: response.data });
+        } else {
+          if (!isAbortedError(response.error)) {
+            setLink({ error: response.error, status: "failed" });
+          }
+        }
+      }
+    },
+    [env, fetchJsonSchemaFromUrl, linkID]
+  );
+
+  useEffect(() => {
+    if (linkID) {
+      const { aborter } = makeRequestAbortable(fetchLink);
+      return aborter;
+    }
+    return;
+  }, [fetchLink, linkID]);
+
+  const loading = isAsyncTaskStarting(link) || isAsyncTaskStarting(credentialSubjectValue);
+
+  const credentialSubjectValueErrorToString = (error: string | z.ZodError) =>
+    error instanceof z.ZodError
+      ? [
+          "An error occurred while parsing the value of the credentialSubject:",
+          ...processZodError(error).map((e) => `"${e}"`),
+        ].join("\n")
+      : `An error occurred while processing the value of the credentialSubject:\n"${error}"`;
+
+  return (
+    <SiderLayoutContent
+      description="View credential link details, attribute values and delete links."
+      showBackButton
+      showDivider
+      title="Credential link details"
+    >
+      {(() => {
+        if (hasAsyncTaskFailed(link)) {
+          return (
+            <Card className="centered">
+              <ErrorResult
+                error={[
+                  "An error occurred while downloading or parsing the link from the API:",
+                  link.error.message,
+                ].join("\n")}
+              />
+            </Card>
+          );
+        } else if (hasAsyncTaskFailed(credentialSubjectValue)) {
+          return (
+            <Card className="centered">
+              <ErrorResult
+                error={credentialSubjectValueErrorToString(credentialSubjectValue.error)}
+              />
+            </Card>
+          );
+        } else if (loading) {
+          return (
+            <Card className="centered">
+              <LoadingResult />
+            </Card>
+          );
+        } else {
+          const { createdAt, expiration, proofTypes, schemaHash, schemaType, status } = link.data;
+
+          const linkURL = `${window.location.origin}${generatePath(ROUTES.credentialLink.path, {
+            linkID,
+          })}`;
+
+          const [flavor, text] = (() => {
+            switch (status) {
+              case "active": {
+                return [{ color: "success", type: "tag" } as const, "Active"];
+              }
+              case "inactive": {
+                return [{ type: "tag" } as const, "Inactive"];
+              }
+              case "exceeded": {
+                return [{ color: "error", type: "tag" } as const, "Exceeded"];
+              }
+            }
+          })();
+
+          return (
+            <Card
+              className="centered"
+              extra={
+                <Button danger icon={<IconTrash />} onClick={() => setShowModal(true)} type="text">
+                  Delete Link
```

Comment:
We don't capitalize "link" or "credential" or "schema" etc. anywhere else so we shouldn't start here.